c++/include/orc/Statistics.hh

```diff
@@ -321,6 +321,32 @@ namespace orc {
      * @return maximum value
      */
     virtual int64_t getMaximum() const = 0;
+
+    /**
+     * check whether column has a lowerBound
+     * @return true if column has a lowerBound
+     */
+    virtual bool hasLowerBound() const = 0;
```

Comment:
I rebase this patch off ORC-87. That is why the changes appear in the overall patch.
I just squashed the commits relevant to only this JIRA. There are two commits now. Once is the ORC-87 commit. The second commit is for this JIRA.